web/src/components/dashboard/ProjectDetails.tsx

```diff
@@ -83,7 +83,7 @@ const ProjectDetails: React.FC<any> = (props) => {
 
   return (
     <>
-      <Box style={{ overflow: 'auto' }}>
+      <Box style={{}}>
```

Comment:
If you remove all style from it, then why we need to keep style props?

Reply:
Right, I have made the changes